osu.Server.Spectator/Hubs/MultiplayerQueue.cs

```diff
@@ -118,7 +103,7 @@ public async Task AddItem(MultiplayerPlaylistItem item, MultiplayerRoomUser user
             if (room.Settings.QueueMode == QueueMode.HostOnly && !user.Equals(room.Host))
                 throw new NotHostException();
 
-            if (room.Playlist.Count(i => i.OwnerID == user.UserID && !i.Expired) >= PER_USER_LIMIT)
+            if (room.Settings.QueueMode != QueueMode.HostOnly && room.Playlist.Count(i => i.OwnerID == user.UserID && !i.Expired) >= PER_USER_LIMIT)
```

Comment:
Hmm, I was thinking the limit would only apply for round robin, but I guess this is okay?